### PR TITLE
Feature/gg 36 custom admin page

### PR DESCRIPTION
**Pull request by orndorffc**
Create custom admin page with improved CSS. Implements Employee, Photo, and Temporary Photo models and creates links which will pull from AD, change authentication options, and add users from temporary photo model when functionality is implemented.

### CODE REVIEW

**Review by lb-cake — COMMENTED** [review]
I approve the changes, but this does create merge conflicts. 

A rebase on development should fix this.

**Comment by mraynor** [comment]
Most of the commit messages are correct, but general rule is to capitalize commit messages

File: GGProject/static/admin/css/changelists.css [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+#changelist {
+    position: relative;
+    width: 100%;
+}
+
+#changelist table {
+    width: 100%;
+}
+
+.change-list .hiddenfields { display:none; }
```
Comment (mraynor) [inline GGProject/static/admin/css/changelists.css]: Looks like most of the other css with one property is on multiple lines, we should be consistent

File: GGProject/static/admin/css/changelists.css [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+    border-radius: 4px;
+    font-size: 14px;
+    padding: 5px;
+    color: #333;
+}
+
+#changelist #toolbar form #searchbar {
+    height: 19px;
+    border: 1px solid #ccc;
+    padding: 2px 5px;
```
Comment (mraynor) [inline GGProject/static/admin/css/changelists.css:91]: Some properties are defined using `em`, while others are defined using `px`. Is there a reason for mixing them?

Comment (orndorffc) [reply GGProject/static/admin/css/changelists.css:91]: These are automatic with django-admin-tools. I am not sure why they are mixed

File: GGProject/static/admin/css/fonts.css [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Roboto';
+    src: url('../fonts/Roboto-Regular-webfont.woff');
+    font-weight: 400;
+    font-style: normal;
+}
+
```
Comment (mraynor) [inline GGProject/static/admin/css/fonts.css:14]: Looks like there are two `@font-face {`, can we combine them?

File: GGProject/static/admin/css/forms.css [hunk 4]
```diff
@@ -0,0 +1,515 @@
...
+    width: 5em;
+}
+
+.vBigIntegerField {
+    width: 10em;
+}
+
+.vForeignKeyRawIdAdminField {
+    width: 5em;
+}
```
Comment (mraynor) [inline GGProject/static/admin/css/forms.css:353]: This has the same styling as `.vIntegerField`, so we can combine them (`.vIntegerField, .vForeignKeyRawIdAdminField {
	    width: 5em;
	}`

File: GGProject/static/admin/css/fonts.css (hunk 3 above)
Comment (orndorffc) [reply GGProject/static/admin/css/fonts.css:14]: It downloads multiple fonts so we should keep them separate.

File: requirements.txt [hunk 5]
```diff
@@ -15,3 +18,4 @@ pytz==2019.1
 requests==2.22.0
 simplejson==3.16.0
 urllib3==1.25.3
+>>>>>>> 33334a727aab38b2014962cb1609515e70ddc8b9
```
Comment (mraynor) [inline requirements.txt]: Looks like we didn't resolve these conflicts

File: GGProject/workflow/tests.py [hunk 6]
```diff
@@ -8,3 +12,4 @@
 
 
 # Place your tests here
+>>>>>>> 33334a727aab38b2014962cb1609515e70ddc8b9
```
Comment (mraynor) [inline GGProject/workflow/tests.py]: Same here

**Review by mraynor — APPROVED** [review]
+1

**Pull request merged**